apps/gitness/src/pages-v2/account/stores/principal-list-store.ts

```diff
@@ -0,0 +1,16 @@
+import { create } from 'zustand'
+
+import { IPrincipalListStore, PrincipalData } from '@harnessio/ui/views'
+
+export const usePrincipalListStore = create<IPrincipalListStore>(
+  (set: (partial: Partial<IPrincipalListStore>) => void) => ({
+    // initial state
```

Comment:
nit: same..we don't need comments initial state/Actions

Reply:
Will cleanup.

Reply:
Done in https://github.com/harness/canary/pull/613.